rdt-open-reader/src/covidIndonesia/assets/json.form/sample-delivery-details-form.json

```diff
@@ -322,5 +288,35 @@
       }
     ]
   },
+  "step5": {
+    "title": "Sample shipment details",
+    "display_back_button": "true",
+    "bottom_navigation": "true",
+    "next_type": "submit",
+    "submit_label": "SUBMIT",
+    "fields": [
+      {
+        "key": "lbl_batch_id",
+        "type": "label",
+        "text_color": "#000000",
+        "label_text_style": "bold",
+        "has_bg": true,
+        "bg_color": "#ffffff",
+        "center_label": "true",
+        "top_margin": "15dp",
+        "text": "Batch Id"
+      },
+      {
+        "key": "batch_id",
+        "type": "label",
+        "text_color": "#000000",
+        "has_bg": true,
+        "bg_color": "#ffffff",
+        "center_label": "true",
+        "top_margin": "15dp",
+        "text": ""
+      }
+    ]
+  },
```

Comment:
Was this generated using JMAG? Some strings don't seem to have been replaced.

Reply:
it was missed, now updated